authentication/migrations/0001_add_token_model.py

```
@@ -0,0 +1,94 @@
+                user_id INTEGER NOT NULL UNIQUE REFERENCES {user_table}(id)
+                    ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED
+            )
+            """
+        )
+        cursor.execute(
+            """
+            INSERT INTO authtoken_token (key, user_id, created)
+            SELECT key, user_id, created
+            FROM authentication_apikey
+            ON CONFLICT (key) DO NOTHING
+            """
```

Ruff (0.14.11)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)